rpc/src/module/net.rs

```diff
@@ -23,7 +23,7 @@ pub(crate) struct NetworkRpcImpl {
 impl NetworkRpc for NetworkRpcImpl {
     fn local_node_info(&self) -> Result<Node> {
         Ok(Node {
-            version: self.network_controller.node_version().to_string(),
```

Comment:
These two writing effects are the same.

ref: https://doc.rust-lang.org/src/alloc/string.rs.html#2122-2132

`to_string` is `format_args!("{}", self)`, the same as `format!("{}", self)`

Reply:
The `Display` impl uses `short`, we changed to use long version instead.

Response before:

    0.12.0

Now:

    0.12.0 (v0.12.0 2019-05-18)

Reply:
Our `Display` is to determine whether `commit_describe` exists. If it exists, it is long. If it does not exist, it is a revision of short. 

That is to say, even if you call long directly, there may be a short display.

Reply:
Confirmed